webapp/config.py

```diff
@@ -34,62 +32,29 @@ def __getattr__(self, name: str) -> Any:
         message = f"'{name}' not a valid configuration variable"
         raise AttributeError(message)
 
-    def _getenv(self, name: str) -> Any:
-        if value := os.getenv(name):
-            return value
-        if name in self.REQUIRED_ENV_VARS:
-            message = f"'{name}' is a required environment variable."
-            raise AttributeError(message)
-        return None
-
     @property
     def ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG(self) -> dict:
-        network_config = os.getenv("ALMA_SAP_INVOICES_ECS_NETWORK_CONFIG")
-        try:
-            return json.loads(network_config)  # type: ignore[arg-type]
-        except (TypeError, JSONDecodeError) as error:
-            raise InvalidNetworkConfigurationError(error) from error
-
-    @property
-    def ALMA_SAP_INVOICES_ECR_IMAGE_NAME(self) -> str:
-        return self._getenv("ALMA_SAP_INVOICES_ECR_IMAGE_NAME")
-
-    @property
-    def ALMA_SAP_INVOICES_ECS_CLUSTER(self) -> str:
-        return self._getenv("ALMA_SAP_INVOICES_ECS_CLUSTER")
-
-    @property
-    def ALMA_SAP_INVOICES_ECS_TASK_DEFINITION(self) -> str:
-        return self._getenv("ALMA_SAP_INVOICES_ECS_TASK_DEFINITION")
-
-    @property
-    def ALMA_SAP_INVOICES_CLOUDWATCH_LOG_GROUP(self) -> str:
-        return self._getenv("ALMA_SAP_INVOICES_CLOUDWATCH_LOG_GROUP")
+        security_groups = self.ALMA_SAP_INVOICES_ECS_GROUPS
+        subnets = self.ALMA_SAP_INVOICES_ECS_SUBNETS
+        return {
+            "awsvpcConfiguration": {
+                "subnets": subnets.split(","),
+                "securityGroups": security_groups.split(","),
```

Comment:
What happens if someone includes a space between the values, e.g. `abc123, def456`?

Personally, I don't think it's worth dedicated properties that would split + strip the values... but felt worth mentioning.  As long as the README outlines the structure of these env vars I think we're okay!

Reply:
`boto3` returns an `InvalidParameterException` for incorrectly formatted parameter values. I agree that formatting errors can be mitigated by a detailed README description for the env var. The descriptions will be updated to: 

```shell
ALMA_SAP_INVOICES_ECS_GROUPS=### Security group(s) for the Alma SAP Invoices ECS task (formatted as a comma-separated string excluding whitespaces, e.g. "sg-abc123,sg-def456").
ALMA_SAP_INVOICES_ECS_SUBNETS=### Subnet(s) for the Alma SAP Invoices ECS task (formatted as a comma-separated string excluding whitespaces, e.g. "subnet-abc123,subnet-def456").
```

Reply:
See [updates in README](https://github.com/MITLibraries/alma-sapinvoices-ui/pull/83/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R30-R31).